ambari-server/src/main/java/org/apache/ambari/server/configuration/AmbariServerConfiguration.java

```diff
@@ -64,5 +73,38 @@ protected String getValue(String propertyName, Map<String, String> configuration
       return defaultValue;
     }
   }
+  
+  /**
+   * @return
```

Comment:
let's remove or populate

Reply:
Will add the missing JavaDoc; thanks for pointing out.